{
    "type": "bundle",
    "id": "bundle--7092d227-6be9-4478-8523-dfa1521802b2",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--417b7752-e4eb-4eff-bc75-f12c0e551ebf",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:12:00.389086Z",
            "modified": "2021-03-13T20:12:00.389086Z",
            "name": "CVE-2019-16874",
            "description": "Portainer before 1.22.1 has Incorrect Access Control (issue 2 of 4).",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-16874"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}